pallets/crowdloans/src/lib.rs

```diff
@@ -1368,5 +1504,11 @@ pub mod pallet {
             ));
             Ok(())
         }
+
+        // Returns contributions for ChildStorageKinds
+        fn get_contributions(vault: &mut Vault<T>, kind: ChildStorageKind) -> u128 {
```

Comment:
dont need to sum, we just need to use `.count` to check if number of contributions is `zero`